asp.in

```diff
@@ -338,8 +338,25 @@ dispatch_action() {
 
 initialize || log_fatal 'failed to initialize asp repository in %s' "$ASPROOT"
 
-while getopts ':a:fhV' flag; do
+while getopts ':a:fhV-:' flag; do
```

Comment:
This is kind of terrible. It means that --arch i686 isn't actually valid, like every other GNU getopts program.

Reply:
yeah, makes. sense. will fix.

Reply:
then which ones would be valid syntax?  --arch=<arch> and / or --arch <arch> ? , and what to do with --arch= <arch>? or --arch =<arch>?